src/sentry/api/endpoints/source_map_debug.py

```diff
@@ -20,6 +20,10 @@
 from sentry.utils.javascript import find_sourcemap
 from sentry.utils.urls import non_standard_url_join
 
+# used to drive logic for when to show the "SDK out of date" error
+JS_VERSION_FOR_DEBUG_ID = "7.44.0"
```

Comment:
@iambriccardo can you confirm the versions pls?

Reply:
@lforst I think this is correct, will need your approval.